app/services/bulk_update/add_trainees/validate_csv.rb

```diff
@@ -18,7 +18,9 @@ def validate!
       attr_reader :csv, :record
 
       def header_row!
-        true
+        return if headers == BulkUpdate::AddTrainees::ImportRows::ALL_HEADERS.keys
```

Comment:
This evaluation will return `false` if the `headers` are in a different order than the `BulkUpdate::AddTrainees::ImportRows::ALL_HEADERS.keys`.